js/calendar.js

```diff
@@ -726,16 +749,20 @@ function notifyTimeToLeave() {
          * How many minutes should pass before the Time-To-Leave notification should be presented again.
          * @type {number} Minutes post the clockout time
          */
-        const notificationInterval = 5;
+        const notificationInterval = getMinutesBetweenNotifications();
         var now = new Date();
         var curTime = String(now.getHours()).padStart(2, '0') + ':' + String(now.getMinutes()).padStart(2, '0');
 
         // Let check if it's past the time to leave, and the minutes line up with the interval to check
         var minutesDiff = hourToMinutes(subtractTime(timeToLeave, curTime));
         var isRepeatingInterval = curTime > timeToLeave && (minutesDiff % notificationInterval === 0);
+        
+        let reachedMaxRepetitions = repetitions === getMaxRepetitionsNumber();
+        let shouldRepeat = !reachedMaxRepetitions && repetitionIsEnabled();
```

Comment:
```suggestion
        let shouldRepeat = !reachedMaxRepetitions && preferences['repetition'];
```
and removal of the ```repetitionIsEnabled()``` function. Because there's no other situation in which it would be needed, the function is superfluous.